packages/features/ee/workflows/components/EventWorkflowsTab.tsx

```diff
@@ -254,15 +258,19 @@ function EventWorkflowsTab(props: Props) {
     <LicenseRequired>
       {!isLoading ? (
         <>
-          {isManagedEventType && (
+          {(isManagedEventType || isChildrenManagedEventType) && (
             <Alert
               severity="neutral"
               className="mb-2"
-              title={t("locked_for_members")}
-              message={t("locked_workflows_description")}
+              title={isChildrenManagedEventType ? t("locked_by_team_admin") : t("locked_for_members")}
```

Comment:
I think it's better to not allow unlocking workflows. I changed the alert: 
<img width="991" alt="Screenshot 2023-10-19 at 14 36 09" src="https://github.com/calcom/cal.com/assets/30310907/1e01e76b-a8af-4b82-9a19-f95c79801267">
<img width="992" alt="Screenshot 2023-10-19 at 14 36 25" src="https://github.com/calcom/cal.com/assets/30310907/e44f4c02-86f1-43eb-8701-9b2908f3413b">